pygmt/helpers/decorators.py

```diff
@@ -439,3 +440,71 @@ def new_module(*args, **kwargs):
         return new_module
 
     return converter
+
+
+def deprecate_parameter(oldname, newname, deprecate_version, remove_version):
+    """
+    Decorator to deprecate a parameter.
```

Comment:
Maybe add an extra note here, feel free to edit my English.

```suggestion
    Decorator to deprecate a parameter.
    
    The old parameter name will be swapped automatically to the new parameter
    name, and users will receive a FutureWarning to inform them of the pending
    deprecation.
```

Reply:
I also added one more note "Use this decorator below the ``use_alias`` decorator.", because `use_alias` will convert long-form parameters to short-form parameters.